### PR TITLE
Change config file finder to only throw when a FodyWeavers.xml file c…

### PR DESCRIPTION
**Pull request by chrischu**
…annot be found on the project and on the solution level

### CODE REVIEW

File: Fody.Tests/ProjectWeaversFinderTests.cs [hunk 1]
```diff
@@ -12,7 +12,9 @@ public void NotFound()
         loggerMock.Setup(x => x.LogDebug(It.IsAny<string>()));
         var logger = loggerMock.Object;
         var testDirectory = TestContext.CurrentContext.TestDirectory;
+        var searchDirectory = Path.Combine(testDirectory, "FodyWeavers.xml");
+
         var weavingException = Assert.Throws<WeavingException>(() => ConfigFileFinder.FindWeaverConfigs(testDirectory, testDirectory, logger));
-        Approvals.Verify(weavingException.Message.Replace(testDirectory, ""));
+        Assert.AreEqual(weavingException.Message, $"Could not find path to weavers file. Searched '{searchDirectory}', '{searchDirectory}'.");
```
Comment (SimonCropp) [inline Fody.Tests/ProjectWeaversFinderTests.cs]: why not approval tests?

Comment (chrischu) [inline Fody.Tests/ProjectWeaversFinderTests.cs]: Is the added complexity really worth it for a single line string? This way the expected result is immediately apparent.

Comment (SimonCropp) [inline Fody.Tests/ProjectWeaversFinderTests.cs]: i much prefer the work flow approval tests. would u mind changing this back

Comment (chrischu) [inline Fody.Tests/ProjectWeaversFinderTests.cs]: Oh somehow missed this comment, I'm gonna get to it asap (probably tonight or tomorrow)!

Comment (chrischu) [inline Fody.Tests/ProjectWeaversFinderTests.cs]: Done!

**Comment by SimonCropp** [comment]
sorry for the delay

**Comment by SimonCropp** [comment]
this should now be on nuget

**Pull request merged**